lxd/device/unix_hotplug.go

```diff
@@ -230,27 +241,33 @@ func (d *unixHotplug) loadUnixDevice() *udev.Device {
 	}
 
 	devices, _ := e.Devices()
-	var device *udev.Device
+	var matchingDevices []udev.Device
 	for i := range devices {
-		device = devices[i]
+		device := devices[i]
 
-		if device == nil {
+		// Ignore USB bus devices (handled by `usb` device type) and devices without a valid devpath or major number.
+		if device == nil || device.Devnum().Major() == 0 || device.Devnode() == "" || strings.HasPrefix(device.Subsystem(), "usb") {
```

Comment:
youve added a comment for the *what* but not yet the *why* of the skipping logic here.

Reply:
the aim here is to document the git archeology you've spent a lot of time on to save future readers the work.